check-formatting.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Pre-commit hook for StartupOS
+# Run it before committing to verify whether all files 
+# are properly formatted so you won't fail early on review
+# Either do it manually or by creating a symlink
+# To do this, execute, from repo root
+# ln -s $(pwd)/pre-commit.sh $(pwd)/.git/hooks/pre-commit
+
+
+npm install &>/dev/null
+
+BUILD_FILES=$(find `pwd` -type f \( -name BUILD.bazel -or -name BUILD \) | grep -v node_modules)
+
+bazel run //tools:buildifier -- -mode=check $BUILD_FILES &>/dev/null || (
```

Comment:
Can you document what `$(tput setaf 1)[!]` and `$(tput sgr0)` do?

Reply:
@oferb yes. I'll restructure it a bit to be self-documenting.